stellarphot/settings/models.py

```diff
@@ -336,15 +336,29 @@ class PhotometryApertures(BaseModelWithTableRep):
 
     radius: Annotated[
         PositiveInt,
-        Field(default=1, json_schema_extra=dict(autoui="ipywidgets.BoundedIntText")),
+        Field(
+            default=1,
+            description="radius of circular annulus, in pixels",
+            json_schema_extra=dict(autoui="ipywidgets.BoundedIntText"),
+        ),
     ]
     gap: Annotated[
         PositiveInt,
-        Field(default=1, json_schema_extra=dict(autoui="ipywidgets.BoundedIntText")),
+        Field(
+            default=1,
+            description="Size of gap between aperture and annulus, in pixels",
+            json_schema_extra=dict(autoui="ipywidgets.BoundedIntText"),
+        ),
     ]
     annulus_width: Annotated[
         PositiveInt,
-        Field(default=1, json_schema_extra=dict(autoui="ipywidgets.BoundedIntText")),
+        Field(
+            default=1,
+            description=(
+                "distance between inner and outer radii of annulus, " "in pixels"
```

Comment:
This line seems malformed, It should work since the parenthesis around the strings effectively collects them together, but WHY was this done here?  The line is not as far as I can see to long to fit within 80 characters.

Reply:
Good catch, it had originally been longer